README.md

```diff
@@ -1,87 +1,77 @@
 # Deployment playbooks for NGI-Pipeline and related software (Piper, TACA, Tarzan, etc.) 
 
+The NGI pipeline is deployed on Irma using ansible playbooks. Ansible playbooks are scripts written for easily adaptable automated deployment. The ansible playbooks are stored here.
 
+## Deployment of the Ansible enviroment
 
-Ansible playbooks to deploy production instances on Irma from irma3. 
+Clone the repository to `/lupus/ngi/irma3/deploy`
 
-## Enabling the functionality of this repo
+Copy the file from the repository under `bootstrap/bashrc` to `/lupus/ngi/irma3/bashrc.` This file has to be sourced every time a user wants to work. 
 
-Checkout this repo to `/lupus/ngi/irma3/deploy`
+Setup a virtual environment, i.e: `/lupus/ngi/irma3/virtualenv-15.0.0/virtualenv.py -p /usr/bin/python2.7 /lupus/ngi/irma3/ansible-env`
 
-Copy the file `bootstrap/bashrc` to `/lupus/ngi/irma3/bashrc`. This file has to be sourced every time a user wants to work. 
+Activate the environment using source `/lupus/ngi/irma3/ansible-env/bin/activate`
 
-Setup a virtual enviroment, i.e: `/lupus/ngi/irma3/virtualenv-15.0.0/virtualenv.py -p /usr/bin/python2.7 /lupus/ngi/irma3/ansible-env`
-
-Activate the environment `source /lupus/ngi/irma3/ansible-env/bin/activate` 
-
-Install ansible `pip install ansible`
+Install ansible with `pip install ansible`
 
 Download Anaconda and install it to `/lupus/ngi/sw/anaconda`
```

Comment:
I think it is important to highlight that all these virtualenv commands, pip commands, anaconda-commands etc needs to be run after bashrc has been sourced, otherwise the umask and gid might end-up incorrect for the installed files. 

It might also be needed to do a manual `chmod -R g+rwX /lupus/ngi/sw/anaconda/` after it has been installed, because it seems the installer don't respect the umask perfectly?